docs/components/movement-sensor/_index.md

```diff
@@ -17,28 +17,23 @@ aliases:
 # SME: Rand
 ---
 
-A movement sensor component is a sensor that gives data on where a machine is and how fast it is moving.
-Examples of movement sensors include global positioning systems (GPS), inertial measurement units (IMUs), accelerometers and gyroscopes.
+If you have a physical sensor such as a GPS, IMU, accelerometer, or gyroscope, or anything else that provides similar measurements, use a _movement sensor_ component.
```

Comment:
This is confusing, as phsyical is much broarder.
Also I would just list out what it does.
`Movement sensors give an API for: gps location, linear velocity and acceleration, angular velocity and acceleration and heading`.